tests/nightly/test_large_array.py

```diff
@@ -19,7 +19,7 @@
 import numpy as np
 import mxnet as mx
 
-from mxnet.test_utils import rand_ndarray, assert_almost_equal, rand_coord_2d, default_context, check_symbolic_forward
+from mxnet.test_utils import rand_ndarray, assert_almost_equal, rand_coord_2d, default_context, check_symbolic_forward, create_2d_tensor
```

Comment:
where do you use default_context?

Reply:
There are 2 tests using this

Reply:
These two tests are not in this PR however. Were they failing earlier than?

Reply:
no